pipeline.yml

```diff
@@ -112,6 +112,10 @@ params:
       TEST_PATH: kubernetes-config/acceptance/redis28
     redis32-tests: &redis32-tests
       SERVICE_NAME: redis32
+      PLAN_NAME: micro
+      TEST_PATH: kubernetes-config/acceptance/redis32
+    redis32-tests: &redis32-tests
```

Comment:
Looks like we're duplicating an anchor here, which could change the behavior of other tests.